### ISSUE TITLE
Uncaught TypeError: Cannot use 'in' operator to search for 'default' in undefined

### ISSUE THREAD

**Issue by impcyber** [yarkovaleksei/vue2-storage#25]
Hi! When i'm trying to use js files from dist directly then i got error:
```
vue2-storage.js:12 Uncaught TypeError: Cannot use 'in' operator to search for 'default' in undefined
    at vue2-storage.js:12
    at vue2-storage.js:9
    at vue2-storage.js:10
```
![image](https://user-images.githubusercontent.com/1425691/53807923-f79ab680-3f61-11e9-9fdc-3dab7ed0a0bd.png)
And there is no matter to download it from this repo or use it from dpkg.

Codepen shows the same error:
https://codepen.io/anon/pen/moOJdY

**Comment by yarkovaleksei** [comment]
@impcyber Thanks for the feedback.
Yes, there really is such a bug, because for some reason the polyfill didn’t turn on when assembling.
I'll fix it the other day.

**Comment by yarkovaleksei** [comment]
Fixed in version 4.0.2
